### PR TITLE
Feature/components

### PR DESCRIPTION
**Pull request by vanapallimanikanta**
added page title header breadcrumb,bussiness card and bussi
Screenshot (55)
Screenshot (56)
Screenshot 2024-09-30 130153
ness card skeleton components

### CODE REVIEW

**Review by Vishnu9901 — CHANGES_REQUESTED** [review]

File: src/libs/ui/src/molecules/BussinessCard/index.tsx [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+    imageSrc: string;
+    title: string;
+    description: string;
+    buttonText: string;
+    onButtonClick: () => void;
+}
+
+const BusinessCard: React.FC<BusinessCardProps> = ({ imageSrc, title, description, buttonText, onButtonClick }) => {
+  return (
+    <div className="w-[500px] h-auto max-w-full lg:border-b ">
```
[the remark this answers is not in the document]
Comment (vanapallimanikanta) [reply src/libs/ui/src/molecules/BussinessCard/index.tsx:16]: w-full was not working as expected; that is the reason I have given w-[500px].By using w-[500px], we can explicitly defining the width to be 500 pixels, which avoids potential issues with relative sizing (w-full)

File: src/libs/ui/src/molecules/BussinessCardSkeleton/index.tsx [hunk 2]
```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+
+const BusinessCardSkeleton: React.FC = () => {
+  return (
+    <div className="w-[500px] h-auto max-w-full rounded-lg overflow-hidden shadow-lg border border-gray-200 animate-pulse">
```
[the remark this answers is not in the document]
Comment (vanapallimanikanta) [reply src/libs/ui/src/molecules/BussinessCardSkeleton/index.tsx:5]: w-full was not working as expected; that is the reason I have given w-[500px].By using w-[500px], we can explicitly defining the width to be 500 pixels, which avoids potential issues with relative sizing (w-full)

File: src/libs/ui/src/molecules/BussinessCard/index.tsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (Vishnu9901) [reply src/libs/ui/src/molecules/BussinessCard/index.tsx:16]: developer can pass custome classes to component

File: src/libs/ui/src/molecules/BussinessCardSkeleton/index.tsx (hunk 2 above)
[the remark this answers is not in the document]
Comment (Vishnu9901) [reply src/libs/ui/src/molecules/BussinessCardSkeleton/index.tsx:5]: developer can pass custom classes to component

[the remark this answers is not in the document]
Comment (vanapallimanikanta) [reply src/libs/ui/src/molecules/BussinessCardSkeleton/index.tsx:5]: okay

**Review by Vishnu9901 — APPROVED** [review]

**Pull request merged**